.travis.yml

```
@@ -9,6 +9,7 @@ julia:
   - 1.0
   - 1.3
   - 1.4
+  - 1.5
```

Let's only test the latest stable release:
```suggestion
  - 1
```

---

done in #295 which is merged here